mlflow/utils/search_utils.py

```diff
@@ -44,8 +46,18 @@ class SearchUtils(object):
         '!=': operator.ne,
         '<=': operator.le,
         '<': operator.lt,
+        'LIKE': re.match,
+        'ILIKE': re.match
     }
 
+    @classmethod
+    def get_sql_filter_ops(cls, column, operator):
+        sql_filter_ops = {
+            'LIKE': column.like,
```

Comment:
What is `column.like`?

Reply:
column.like is the way we perform the like operation with sqlalchemy. (column being a column of a sqlalchemy object that we pass as parameter here)